Abstract Syntax and Semantics Document/index.html

```diff
@@ -156,40 +156,57 @@
-    
+    <section id="def-timestamped-graph">
     <h3>Timestamped graph</h3>
     
-    <p>A <em>timestamped graph</em> is defined as an RDF Dataset under the RDF Dataset semantics that <a href="http://www.w3.org/TR/2014/NOTE-rdf11-datasets-20140225/#each-named-graph-defines-its-own-context">each graph defines its own context</a> with the following constraints.</p>
+      <p>A <em>timestamped graph</em> is defined as an RDF Dataset under the RDF Dataset semantics that <a href="http://www.w3.org/TR/2014/NOTE-rdf11-datasets-20140225/#each-named-graph-defines-its-own-context">each graph defines its own context</a> and where a particular triple in the default graph has been designated as the timestamp triple, with the following constraints.</p>
     
     <ol>
-      <li>There is a single named graph pair (n, g) in the RDF Dataset (where <code>g</code> is an <a href="http://www.w3.org/TR/rdf11-concepts/#section-rdf-graph">RDF graph</a>, and <code>n</code> is an IRI or blank node).</li>
-      <li>There is a <del>single</del> triple in the default graph of the RDF Dataset, and it has the form (n, p, t), where <code>n</code> is defined in the previous item, <code>p</code> is a predicate that captures the relationship between the temporal entity <code>t</code>, called the timestamp, and the graph <code>g</code>.</li>
+      <li>There is exactly one named graph pair <code>&lt;n, G></code> in the RDF Dataset (where <code>G</code> is an <a href="http://www.w3.org/TR/rdf11-concepts/#section-rdf-graph">RDF graph</a>, and <code>n</code> is an IRI or blank node).</li>
+      <li>The timestamp triple has the form <code>&lt;n, p, t></code>, where <code>n</code> is defined in the previous item, <code>p</code> is a timestamp predicate that captures the relationship between the temporal entity <code>t</code>, called the timestamp, and the graph <code>G</code>.</li>
     </ol>
-    
-    <p>Limitations of the definition:</p>
-    
-    <ul>
-      <li>does <strong>not</strong> allow the default graph of a timestamped graph to have more than triple.</li>
-    </ul>
-    
-    <blockquote>
-      <p><strong>Discussion:</strong> There could be multiple timestamps associated with a graph <code>g</code>, e.g. a start time and an end time, or a generated time and a system processing time. The predicate <code>p</code> should be drawn from a community agreed vocabulary (<a href="https://github.com/streamreasoning/RSP-QL/issues/10">Issue 10</a>).</p>
+      <p class="note">There may be multiple triples in the default graph, including multiple triples using timestamp predicates, 
+        but exactly one triple must be designated as the timestamp triple.</p>
+    
+    <p class="ednote">This definition does not permit the timestamp to be omitted, which is one of the data structures
+      that is considered to be in-scope by the requirements document.
+    </p>
+      <p class="note">
+      This definition allows the timestamp to be implicit.
```

Comment:
This definition allows the timestamp to be implicit.
=>
This definition allows the timestamp to be an IRI or blanknode. Additional information may be provided about it in the default graph (e.g. through properties in the OWL Time Ontology), but this is not required by the definition of timestamped graph.